tools/ci/production-verify

```diff
@@ -56,6 +56,9 @@ if supervisorctl status | grep -vq RUNNING || supervisorctl status | sed 's/^.*u
     echo
     echo "DEBUG: printing Zulip server's tornado log:"
     cat /var/log/zulip/tornado.log
+    echo
+    echo "DEBUG: printing process_fts_updates log:"
+    cat /var/log/zulip/fts-updates.log
```

Comment:
This should be its own mini commit, since it's useful debugging details.

Reply:
This has been changed.